config/crd/bases/infrastructure.cluster.x-k8s.io_ocimanagedclusters.yaml

```diff
@@ -108,6 +108,7 @@ spec:
                     description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
                     type: string
                 type: object
+                x-kubernetes-map-type: atomic
```

Comment:
more for my own learning: What is setting these new server-side apply fields?

Reply:
not sure why it did not add earlier and added now when we moved the spec, it is present in OCICluster identityRef earlier itself https://github.com/oracle/cluster-api-provider-oci/blob/main/config/crd/bases/infrastructure.cluster.x-k8s.io_ociclusters.yaml#L108